cloud/corfu/cluster_deploy.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/zsh
```

Comment:
Just some thoughts:
When the sh script is getting complex you should switch to a better tool. 
Maybe using python could be a better solution maybe some other autmation tool, like terraform.

Just to note, if you'd like to replace this sh with something else in the future, I'd be happy to support that activity